config/core/resources/eventtype.yaml

```diff
@@ -20,6 +20,15 @@ metadata:
     knative.dev/crd-install: "true"
 spec:
   group: eventing.knative.dev
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      # this is a work around so we don't need to flush out the
+      # schema for each version at this time
+      #
+      # see issue: https://github.com/knative/serving/issues/912
+      x-kubernetes-preserve-unknown-fields: true
```

Comment:
Since you have a schema int he v1alpha1 this should be moved to the `v1beta1` validation.

Since you cannot have a top-level validation (applies to all versions) & per-version validation at the same time

Reply:
Just remove it from v1alpha1 as per other PRs.